OpenSim/Common/ComponentSocket.h

```diff
@@ -884,15 +884,15 @@ class Input : public AbstractInput {
     /** @{                                                               */ \
     /** comment                                                          */ \
     /** In an XML file, you can set this Socket's connectee name         */ \
-    /** via the <b>\<socket_##cname##_connectee_name\></b> element.      */ \
+    /** via the <b>\<socket_##cname\></b> element.      */ \
```

Comment:
Can you realign the end of this line (here and below)?

Reply:
Done